tests/installation/bootloader_uefi.pm

```diff
@@ -57,6 +57,11 @@ sub run {
         return;
     }
 
+    # press key up to cancel 10 sec timeout on jeos images to don't miss the grub screen
```

Comment:
Unrelated change?

Reply:
This is failure which were blocking VR, it has separate commit.
I didn't see any poo so i just added it to this PR.

Reply:
IMO this needs a separate discussion. @mloviska has a similiar hack in tests/jeos/grub2.pm already, so adding a duplicate here is not great.

Reply:
I can see that the community JeOS ( aarch64 ) still uses `bootloader_uefi.pm`. 
Not sure, how to proceed over here. Whether to put the grub2.pm into generic lib or if we want to modify the schedule. WDYT @Vogtinator, @dzedro

Reply:
I will drop this then and create new PR

Reply:
@mloviska Is there any reason that JeOS can't use the generic bootloader modules? It would IMO be the best option here if we can avoid having a specific `tests/jeos/grub2.pm`.

Reply:
dropped, I will crate separate PR for it.

Reply:
> @mloviska Is there any reason that JeOS can't use the generic bootloader modules? It would IMO be the best option here if we can avoid having a specific `tests/jeos/grub2.pm`.

Images have reduced timeout, compared to isos at first boot stage. Race condition happens at this point, in case we want to interact with grub to set gfxpayload https://openqa.suse.de/tests/4238030#step/grub2/3. 
But as you already pointed out, it is not necessary to set `gfxpayload` when we use `virtio` driver. 
http://kepler.suse.cz/tests/29#step/console_reboot/11